CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## Upcoming...
 
 - ... <!-- Add new lines here. -->
+- **BREAKING** feat: Update eslint to version 7 and prettier to version 2 – which do not work in Node@8
+- feat: Update optional/peer dependencies to latest versions
```

Comment:
- **BREAKING** feat: Update `pkg.engines.node` version to `^10.12`
- feat: Update all dependencies to latest versions